src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,17 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: $transition-setting;
+  
+
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
   }
+
+  &__card:hover {
+    transform: scale(1.2);
+    color: #34568b;
```

Comment:
This transition is also not happening smoothly due to the fack I described in prev comment